src/algorithms/calorimetry/CalorimeterClusterRecoCoG.cc

```diff
@@ -175,6 +175,7 @@ std::optional<edm4eic::Cluster> CalorimeterClusterRecoCoG::reconstruct(const edm
     m_log->debug("hit energy = {} hit weight: {}", hit.getEnergy(), weight);
     auto energy = hit.getEnergy() * weight;
     totalE += energy;
+    cl.addToHits(hit);
```

Comment:
We also have `hitContributions` in [`edm4eic::Cluster`](https://github.com/eic/EDM4eic/blob/main/edm4eic.yaml#L258). Could we fill that with `energy` here too?

Reply:
An interesting question is also whether it makes more sense to fill the `energy` or the weight that goes into the mean position determination... We can't really store both. Are they even different?